apps/aecore/src/aec_subscribe.erl

```diff
@@ -24,7 +24,7 @@
 
 -type id() :: {ws, ws_handler:id()}.
 -type aeo_event() :: {query, aeo_oracles:id()}
-                   | {response,  aeo_oracles:id(), aeo_interaction:id()}.
+                   | {response, aeo_query:id()}.
```

Comment:
I understand oracle query id includes oracle id.

Reply:
Not really, this is just a proof of how utterly useless Dialyzer is at some things. That type was blatantly wrong before, so this is a renaming _and_ a spec-fix. (In fact if you remember the list comprehension you asked me to change yesterday, this was the incorrect type-spec for it...)